input/collector_platform.go

```diff
@@ -0,0 +1,36 @@
+package input
+
+import (
+	"runtime"
+	"time"
+
+	"github.com/pganalyze/collector/state"
+	"github.com/shirou/gopsutil/host"
+)
+
+// TODO: is there a better place to initialize this?
+var collectorStartTime = time.Now()
```

Comment:
Also, this causes a test failure in the transform test because it introduces non-determinism. We can potentially just zero it out before comparing, but not sure if this is the right approach in the first place.

Reply:
We could put this into the `state.CollectionOpts` and initialize this in the main function?

(and for tests, passing the collection opts would allow us to set a specific start time and then verify it gets passed through)

Reply:
That makes sense :+1: